packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2370,8 +2359,14 @@ abstract class BuildContext {
 /// {@end-tool}
 class BuildOwner {
   /// Creates an object that manages widgets.
+  ///
+  /// If the `focusManager` argument is not specified, this will construct a
```

Comment:
nit: maybe repeat some of these docs on the focusManager field (e.g. what this defaults to)?